packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -32,7 +35,52 @@ const Color _kDefaultGlowColor = Color(0xFFFFFFFF);
 @immutable
 class ScrollBehavior {
   /// Creates a description of how [Scrollable] widgets should behave.
-  const ScrollBehavior();
+  const ScrollBehavior({
+    @Deprecated(
+      'Temporary migration flag, do not use. '
+      'This feature was deprecated after v2.1.0-11.0.pre.'
+    )
+    bool useDecoration = false,
+    this.scrollbarPlatforms,
+    this.glowingPlatforms,
+  }) : _useDecoration = useDecoration;
+
+  ///
+  ScrollBehavior copyWith({
+    Set<TargetPlatform>? scrollbarPlatforms,
+    Set<TargetPlatform>? glowingPlatforms,
+  }) {
+    return ScrollBehavior(
+      scrollbarPlatforms: scrollbarPlatforms ?? this.scrollbarPlatforms ?? ScrollBehavior.defaultScrollbarPlatforms,
+      glowingPlatforms: glowingPlatforms ?? this.glowingPlatforms ?? ScrollBehavior.defaultGlowingPlatforms,
+      useDecoration: true,
+    );
+  }
+
+  // Whether [buildViewportChrome] or [buildViewportDecoration] should be used
+  // in wrapping the Scrollable widget.
+  //
+  // This is used to maintain subclass behavior to allow for graceful migration.
+  final bool _useDecoration;
+
+  ///
+  final Set<TargetPlatform>? scrollbarPlatforms;
+
+  ///
+  static Set<TargetPlatform> get defaultScrollbarPlatforms => <TargetPlatform>{
```

Comment:
Instead of creating a new set every time this is called, can this just be a static const set?